Library/Homebrew/cmd/update.sh

```diff
@@ -343,51 +350,51 @@ EOS
   fi
 
   # check permissions
-  if [[ -e "$HOMEBREW_CELLAR" && ! -w "$HOMEBREW_CELLAR" ]]
+  if [[ -e "${HOMEBREW_CELLAR}" && ! -w "${HOMEBREW_CELLAR}" ]]
   then
     odie <<EOS
-$HOMEBREW_CELLAR is not writable. You should change the
-ownership and permissions of $HOMEBREW_CELLAR back to your
+${HOMEBREW_CELLAR} is not writable. You should change the
+ownership and permissions of ${HOMEBREW_CELLAR} back to your
 user account:
-  sudo chown -R \$(whoami) $HOMEBREW_CELLAR
+  sudo chown -R \$(whoami) ${HOMEBREW_CELLAR}
 EOS
   fi
 
-  if [[ ! -w "$HOMEBREW_REPOSITORY" ]]
+  if [[ ! -w "${HOMEBREW_REPOSITORY}" ]]
   then
     odie <<EOS
-$HOMEBREW_REPOSITORY is not writable. You should change the
-ownership and permissions of $HOMEBREW_REPOSITORY back to your
+${HOMEBREW_CELLAR} is not writable. You should change the
+ownership and permissions of ${HOMEBREW_CELLAR} back to your
 user account:
-  sudo chown -R \$(whoami) $HOMEBREW_REPOSITORY
+  sudo chown -R \$(whoami) ${HOMEBREW_CELLAR}
```

Comment:
Is there a reason for changing this from `HOMEBREW_REPOSITORY` to `HOMEBREW_CELLAR`?

Reply:
I made a mistake, and I will fix it soon.

Reply:
No worries - just checking! ❤️